warehouse/query-core/src/test/java/datawave/query/jexl/visitors/HasUnfieldedTermVisitorTest.java

```diff
@@ -2,56 +2,58 @@
+
 public class HasUnfieldedTermVisitorTest {
     
-    private static final Logger log = ThreadConfigurableLogger.getLogger(HasUnfieldedTermVisitorTest.class);
-    
-    String[] originalQueries = { //
-    
-    "_ANYFIELD_ == 'FOO'", //
-            "AG.max() == 40", //
-            "BIRTH_DATE.min() < '1920-12-28T00:00:05.000Z'", //
-            "FOO == 'bar'"};
-    boolean[] expectedResults = {true, false, false, false};
+    private ASTJexlScript script;
     
     @Test
     public void test() throws Exception {
+        givenJexlQuery("_ANYFIELD_ == 'FOO'");
+        assertTrue(hasUnfieldedTermVisitor());
         
-        for (int i = 0; i < originalQueries.length; i++) {
-            String query = originalQueries[i];
-            ASTJexlScript script = JexlASTHelper.parseJexlQuery(query);
-            
-            Assert.assertEquals("query:" + query, expectedResults[i],
-                            ((AtomicBoolean) new JexlASTHelper.HasUnfieldedTermVisitor().visit(script, new AtomicBoolean(false))).get());
-        }
+        givenJexlQuery("AG.max() == 40");
+        assertFalse(hasUnfieldedTermVisitor());
+        
+        givenJexlQuery("BIRTH_DATE.min() < '1920-12-28T00:00:05.000Z'");
+        assertFalse(hasUnfieldedTermVisitor());
+        
+        givenJexlQuery("FOO == 'bar'");
+        assertFalse(hasUnfieldedTermVisitor());
     }
     
     @Test
     public void testLucene() throws Exception {
+        givenLuceneQuery("FOO");
+        assertTrue(hasUnfieldedTermVisitor());
+        
+        givenLuceneQuery("FOO || BAR || AG:40");
+        assertTrue(hasUnfieldedTermVisitor());
         
-        String[] originalQueries = { //
+        givenLuceneQuery("BIRTH_DATE:1920");
+        assertFalse(hasUnfieldedTermVisitor());
         
-        "FOO", //
-                "FOO || BAR || AG:40", //
-                "BIRTH_DATE:1920", //
-                "FOO:bar"};
-        boolean[] expectedResults = {true, true, false, false};
-        for (int i = 0; i < originalQueries.length; i++) {
-            String query = originalQueries[i];
-            ASTJexlScript script = JexlASTHelper.parseJexlQuery(new LuceneToJexlQueryParser().convertToJexlNode(query).toString());
-            // JexlASTHelper.parseJexlQuery(query);
-            
-            Assert.assertEquals("query:" + query, expectedResults[i],
-                            ((AtomicBoolean) new JexlASTHelper.HasUnfieldedTermVisitor().visit(script, new AtomicBoolean(false))).get());
-        }
+        givenLuceneQuery("FOO:bar");
+        assertFalse(hasUnfieldedTermVisitor());
     }
     
+    private void givenJexlQuery(String query) throws ParseException {
+        script = JexlASTHelper.parseJexlQuery(query);
+    }
+    
+    private void givenLuceneQuery(String query) throws datawave.query.language.parser.ParseException, ParseException {
+        script = JexlASTHelper.parseJexlQuery(new LuceneToJexlQueryParser().convertToJexlNode(query).toString());
```

Comment:
Could be simplified to 

`givenJexlQuery(new LuceneToJexlQueryParser().convertToJexlNode(query).toString())`